src/support/timestamp.c

```diff
@@ -245,6 +245,57 @@ __wt_time_aggregate_validate(
 {
     char time_string[2][WT_TIME_STRING_SIZE];
 
+    /*
+     * The aggregated time window values that are tracked at the page level.
+     *    newest_start_durable_ts - The default value is WT_TS_NONE. It tracks the maximum durable
+     timestamp of all the inserts performed on a page.
+     *    newest_stop_durable_ts - The default value is WT_TS_NONE. It tracks the maximum durable
+     timestamp of all the removes performed on a page.
```

Comment:
replace "removes" with " the delete operations"

Reply:
Fixed in d0719a967c76f22f12ccd16c910bb1b8acd8522b